src/cpp/src/text2image/stable_diffusion_pipeline.hpp

```diff
@@ -161,7 +161,7 @@ class Text2ImagePipeline::StableDiffusionPipeline : public Text2ImagePipeline::D
             generation_config.random_generator = std::make_shared<CppStdGenerator>(seed);
         }
 
-        ov::Tensor encoder_hidden_states = m_clip_text_encoder->infer(positive_prompt, generation_config.negative_prompt,
+        ov::Tensor encoder_hidden_states = m_clip_text_encoder->infer(positive_prompt, *generation_config.negative_prompt,
```

Comment:
do we need to check that `generation_config.negative_prompt` is not nullopt ?

BTW, do we need to make `negative_prompt` to be optional? looks like only `negative_prompt_2` and `negative_prompt_3` should be optional as we need to distinguish when they are specified as empty string or not specified at all (and should be assigned to `negative_prompt`)